test/e2e/e2e_test.go

```diff
@@ -44,11 +45,11 @@ var _ = ginkgo.Describe("TrainJob e2e", func() {
 	})
 
 	// These tests create TrainJob that reference supported runtime without any additional changes.
-	ginkgo.When("creating TrainJob", func() {
+	ginkgo.When("should succeed to perform the PyTorch workload", func() {
```

Comment:
Isn't the best practice to have logically correct sentence with `ginkgo.When`  ?
Like in Kueue: https://github.com/kubernetes-sigs/kueue/blob/main/test/e2e/multikueue/e2e_test.go#L204
```suggestion
	ginkgo.When("creating a TrainJob to perform the PyTorch workload", func() {
```

Reply:
Ah, yes. The top level `When` is used to construct the head of error message.

Reply:
Done.